packages/frontend-api/src/Model/Mutation/Cart/CartMutation.php

```diff
@@ -135,4 +138,31 @@ public function addOrderItemsToCartMutation(
 
         return $cartWithModificationsResult;
     }
+
+    /**
+     * @param \Overblog\GraphQLBundle\Definition\Argument $argument
+     * @param \Overblog\GraphQLBundle\Validator\InputValidator $validator
+     * @throws \Overblog\GraphQLBundle\Validator\Exception\ArgumentsValidationException
+     * @return array
+     */
+    public function removeCartMutation(Argument $argument, InputValidator $validator): array
+    {
+        $validator->validate();
+        $input = $argument['input'];
+
+        if (array_key_exists('cartUuid', $input) && $input['cartUuid'] !== null) {
+            $cart = $this->cartApiFacade->findCart(null, $input['cartUuid']);
+        } else {
+            $customerUser = $this->currentCustomerUser->findCurrentCustomerUser();
+            $cart = $this->cartApiFacade->findCart($customerUser, null);
+        }
```

Comment:
How does it work when a logged customer calls this mutation with some `cartUuid`?